nucliadb_utils/nucliadb_utils/settings.py

```diff
@@ -125,6 +125,10 @@ class IndexingSettings(BaseSettings):
     index_jetstream_auth: Optional[str] = None
     index_local: bool = False
 
+    indexed_jetstream_target: str = "indexed.{partition}"
```

Comment:
Shouldn't these settings be on the node? Maybe this whole settings class should be moved there?

Reply:
yes I agree with you. To not make the dependency between node and utils worse, I will isolate the settings in the node

Reply:
done